addons/crm/models/utm.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api, SUPERUSER_ID
+
+class UtmCampaign(models.Model):
+    # OLD crm.case.resource.type
+    _inherit = 'utm.campaign'
+    _description = 'UTM Campaign'
```

Comment:
Don't add a new description when inheriting, we should have it just at first model declaration. It is easier to update that way when necessary.